roles/disk/README.rst

```diff
@@ -0,0 +1,9 @@
+Disk
```

Comment:
Why make a new role rather than putting this in with the LVM role? They seem very interdependent, whereas I think of roles as (ideally) being at least partially modular/orthogonal.